inc/ses_to_cloudwatch/namespace.php

```diff
@@ -7,8 +7,8 @@
 
 namespace Altis\Cloud\SES_To_CloudWatch;
 
-use Altis\Cloud\CloudWatch_Logs;
 use Exception;
+use function Altis\Cloud\log_to_cloud;
```

Comment:
I think we disallowed `use function` in recent coding standards. You should always `use Altis\Cloud` and then call `Cloud\log_to_cloud`